Jenkinsfile

```diff
@@ -85,7 +85,7 @@ pipeline {
                     steps {
                         checkPatch user: GITHUB_USER_USR,
                                    password: GITHUB_USER_PSW,
-                                   ignored_files: "src/control/vendor/*"
+                                   ignored_files: "src/control/vendor/*:src/mgmt/*.pb-c.[ch]"
```

Comment:
this was suggested by @brianjmurrell to avoid unintentionally ignoring files that might coincidentally share a similar suffix in future

Reply:
@dpquigl: As discussed with @tanabarr, overly-broad pattern matching just gives me pause for concern about unintentionally matching things that were never imagined would be matched when the pattern is established.

Particularly when those patterns are being used to exclude things from inspection they can lead to long-standing false positive situations and be a bear to uncover once a long-standing false positive is finally discovered.

But I also said that ultimately, it's just advise and @tanabarr can make the pattern whatever he wants.  Ultimately it's just my personal preference when writing patterns to try to be as conservative as I can, in balance with the number and complexity of patterns I might need to make, etc.

Reply:
I'm fine with making the changes specific as suggested by @brianjmurrell

Reply:
IMO the *.pb-c.[ch] suffix is unique enough to serve as a flag for when a file is a generated protobuf file. But I'm fine either way.

Reply:
I agree with Kris. No other person is going to create *.pb-c.* files except the protoc compiler and we should never be running code style checks on generated source code. The problem with specifying it the way Tom did is we should have at least 4 paths in there then. Ton's management protoc, the drpc protoc, the security protos, and probably all the paths for the test file protos as well. We also don't know how many more people will be using dRPC (it seems Li Wei is creating some dRPC calls as well) so that is yet another path which will have to be ignored. I appreciate that you don't want overly broad patterns but it seems justified in this case.